### PR TITLE
fix docker packages from name change

### PR DESCRIPTION
**Pull request by ChawinTan**
### [Describe your changes]

Rename docker packages to `rmf-web` after name change from `romi-dashboard`

## Self-checks

- [ ] I'm familiar with and follow this [ Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [x] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

## Discussion

[Questions for reviewers]

### CODE REVIEW

File: packages/dashboard/docker/e2e.Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM docker.pkg.github.com/osrf/romi-dashboard/rmf:nightly
+FROM docker.pkg.github.com/osrf/rmf-web/e2e/rmf:nightly
```
Comment (koonpeng) [inline packages/dashboard/docker/e2e.Dockerfile:1]: This doesn't look right. Shouldn't it be
```suggestion
FROM docker.pkg.github.com/osrf/rmf-web/rmf:nightly
```

File: packages/dashboard/docker/rmf/docker-compose.yml [hunk 2]
```diff
@@ -2,4 +2,4 @@ version: "3"
 services:
   build:
     build: .
-    image: docker.pkg.github.com/osrf/romi-dashboard/rmf:nightly
+    image: docker.pkg.github.com/osrf/rmf-web/e2e/rmf:nightly
```
Comment (koonpeng) [inline packages/dashboard/docker/rmf/docker-compose.yml:5]: This doesn't look right. Shouldn't it be
```suggestion
FROM docker.pkg.github.com/osrf/rmf-web/rmf:nightly
```

File: packages/dashboard/e2e/README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Below is a diagram of how the containers and networks interact when up and runni
 
 Broadly speaking, there are 3 major parts of the system:
 
-- Romidashboard container, built from the `docker.pkg.github.com/osrf/romi-dashboard/e2e` image with the RMF backend compiled into it. It holds the ROS2 bridge server and the React dashboard. Sourcing of RMF is required before running the tests and it can be sourced from `. /opt/rmf/setup.bash`
+- Romidashboard container, built from the `docker.pkg.github.com/osrf/rmf-web/e2e/e2e` image with the RMF backend compiled into it. It holds the ROS2 bridge server and the React dashboard. Sourcing of RMF is required before running the tests and it can be sourced from `. /opt/rmf/setup.bash`
```
Comment (koonpeng) [inline packages/dashboard/e2e/README.md:22]: This doesn't look right. Shouldn't it be `docker.pkg.github.com/osrf/rmf-web/e2e`?

**Review by koonpeng — CHANGES_REQUESTED** [review]

**Review by koonpeng — APPROVED** [review]
thanks for these, :crossed_fingers: for nightly builds
